examples/mplot3d/spherical_polygon.py

```diff
@@ -0,0 +1,28 @@
+from mpl_toolkits.mplot3d import proj3d
+from scipy.spatial import ConvexHull
```

Comment:
My apologies. I can calculate this specific triangulation myself and just supply it manually. (In general this is of course not so easy.)